test/blackbox-tests/test-cases/melange/include_subdirs.t/run.t

```diff
@@ -0,0 +1,6 @@
+Test that libs using `(include_subdirs unqualified) work well with `melange.emit` stanza
+
+Build js files
+  $ dune build inside/output/app/app__B.js
```

Comment:
it'd be good to unmangle these and produce `app.js`

Reply:
I am not sure why (I don't understand well how wrapped libs are implemented in Dune) but if I build `app.js` I get some empty module:

```js
// Generated by Melange
'use strict';


var App = {};

exports.App = App;
/* No side effect */
```

Not sure why it is missing `B`?